### DISCUSSION TITLE
Logdata / plantname+date

### DISCUSSION THREAD

**Discussion by shadowbyte68** [SBFspot/SBFspot#489] [Q&A] [answered]
Hello,

I want to reduce log data from sbfspot. I have found log files with "Plant name" + "Date" + ". Log"
These are updated every 5 minutes.

Where can I deactivate this?

**Comment by sbf-** [comment, marked as answer]
In `/usr/local/bin/sbfspot.3/daydata` change the line `log=/var/log/sbfspot.3/PlantName_$(date '+%Y%m%d').log` to `log=/dev/null`

**Reply by shadowbyte68** [reply]
Thanks !

**Comment by shadowbyte68** [comment]
#!/bin/bash
log=/dev/null
/usr/local/bin/sbfspot.3/SBFspot -v -ad1 -am0 -ae1 -mqtt -cfg/usr/local/bin/sbfspot.3/SBFspot.cfg $1 $2 $3 $4 $5 &>>$log

It dosent't work since SBFspot-update ...

**Reply by sbf-** [reply]
... since upgrade to wich version?

**Reply by shadowbyte68** [reply]
SBFspot 3.9.4
